generator/src/main/java/org/apache/kafka/message/checker/MetadataSchemaCheckerTool.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.message.checker;
+
+import net.sourceforge.argparse4j.ArgumentParsers;
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+import net.sourceforge.argparse4j.inf.Namespace;
+import net.sourceforge.argparse4j.inf.Subparser;
+import net.sourceforge.argparse4j.inf.Subparsers;
+import net.sourceforge.argparse4j.internal.HelpScreenException;
+
+import java.io.PrintStream;
+
+public class MetadataSchemaCheckerTool {
+    public static void main(String[] args) throws Exception {
+        try {
+            run(args, System.out);
+        } catch (HelpScreenException e) {
+        }
+    }
+
+    public static void run(
+        String[] args,
+        PrintStream writer
+    ) throws Exception {
+        ArgumentParser argumentParser = ArgumentParsers.
+            newArgumentParser("metadata-schema-checker").
+            defaultHelp(true).
+            description("The Kafka metadata schema checker tool.");
+        Subparsers subparsers = argumentParser.addSubparsers().dest("command");
+        Subparser parseParser = subparsers.addParser("parse").
+            help("Verify that a JSON file can be parsed as a MessageSpec.");
+        parseParser.addArgument("--path", "-p").
+            required(true).
+            help("The path to a schema JSON file.");
+        Subparser evolutionVerifierParser = subparsers.addParser("verifyEvolution").
```

Comment:
Should this be `verify-evolution`? I think we use dagger-case in other CLIs (like `kafka-storage.sh random-uuid`)

Reply:
yeah, let's use dashes here